auth-web/package-lock.json

```diff
@@ -17343,6 +17343,7 @@
       "version": "1.4.0",
       "inBundle": true,
       "license": "MIT",
+      "optional": true,
```

Comment:
I think there's something funny going on here. Did you do a straight `npm install` or something else?

Also please confirm that you're using node v16.

Reply:
Yes, I did do straight `npm install`.

Reply:
@lambert-alex , you saw this once, didn't you? Do you remember what the issue was?

@lewischenstudio , what version of Node are you using?

Reply:
I'm using Node 16

![image](https://user-images.githubusercontent.com/51458187/199294057-5b37911c-5af8-4053-b911-91eca5701ef3.png)

Reply:
OK... I don't why npm added these `optional` fields, but now it's doing it with me, too.

I also note that there are 129 other instances of this field in here, so this is probably a normal thing.

Therefore, I accept this package-lock.json file change.

Reply:
I found this, which seems to indicate this is not a problem:

https://stackoverflow.com/questions/54411377/package-lock-json-file-package-with-optional-true

For the record, I am using npm 8.13.2.

Interestingly, if I do `npm ci` (clean install), the additional optional fields are not generated. Maybe you could try that, so that you only commit the app version change?

Reply:
Thanks. Good to know.

Can you add this symptom and what you know about it to a new discussion here please? (For the next time we see this.)
https://github.com/bcgov/lear/discussions

Reply:
@severinbeauvais @lewischenstudio are you guys still having issues? I am not. If it is similar to my problem you should be able to reset your npm config file to fix it.

Reply:
No, the extra fields were not added when I did npm ci.

Reply:
I did `npm ci`, but it doesn't update the package version in package-lock.json file from package.json.

Reply:
Sorry I meant when using npm i